src/components/popup/details/vendors/vendors.jsx

```diff
@@ -42,11 +42,21 @@ export default class Vendors extends Component {
 		});
 	};
 
+	getActiveAttributesNameElements = (setOfAttributes, idsOfActiveAttributes, translationPrefix='') => {
+		const activeAttributes = setOfAttributes
+			.filter(attribute => idsOfActiveAttributes.includes(attribute['id']))
```

Comment:
Unfortunately `Array.prototype.includes` doesn't work in Internet Explorer. Please use `Array.prototype.indexOf` or something else that is more widely supported.